testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -363,6 +363,23 @@ def test_global(self, universe):
             "resname LYS and name NZ and around 4 backbone")
         ag2 = ag.select_atoms("around 4 global backbone")
         assert_equal(ag2.indices, ag1.indices)
+    
+    @pytest.mark.parametrize('selstr',
+        ["resname LYS and name NZ",
+        "protein", "backbone", "resid 100",
+        "resid 100:105",
+        "around 4.0 bynum 1943"])
+    def test_accept_atomgroup(self, universe, selstr):
+        ag = universe.select_atoms(selstr)
+        assert ag is universe.select_atoms(ag)
+    
+    @pytest.mark.parametrize('selstr',
+        ["around 4.0 bynum 1943",
```

Comment:
The around statements are gonna be slow, especially if this is a real universe. Technically any selection (eg on names) with updating will be updating.

Reply:
Changed the selection string to 'resid 100'